src/connector/src/sink/iceberg/mod.rs

```diff
@@ -97,6 +97,103 @@ use crate::sink::{
 use crate::{deserialize_bool_from_string, deserialize_optional_string_seq_from_string};
 
 pub const ICEBERG_SINK: &str = "iceberg";
+
+// ============ Helper functions for schema change ============
+
+/// Convert Arrow `DataType` to Iceberg `Type`
+fn arrow_type_to_iceberg_type(arrow_type: &ArrowDataType) -> Result<iceberg::spec::Type> {
```

Comment:
Can we use these method in the sdk instead? https://github.com/risingwavelabs/iceberg-rust/blob/4eafd2ca22e032c3ee4d9e6abe6ff3eae93e0189/crates/iceberg/src/arrow/schema.rs#L214-L228